landoui/server.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import click
+
+from flask import Flask
+
+
+@click.command()
+@click.option(
+    '--run-dev-server', envvar='RUN_DEV_SERVER', type=bool, default=False
+)
+@click.option('--debug', envvar='DEBUG', type=bool, default=False)
+@click.option('--port', envvar='PORT', type=int, default=80)
+@click.option('--host', envvar='HOST', default='0.0.0.0')
+@click.option(
+    '--version-path', envvar='VERSION_PATH', default='/app/version.json'
+)
+@click.option('--secret-key', envvar='SECRET_KEY', default=None)
+@click.option(
+    '--session-cookie-name', envvar='SESSION_COOKIE_NAME', default=None
+)
+@click.option(
+    '--session-cookie-domain', envvar="SESSION_COOKIE_DOMAIN", default=""
+)
+@click.option(
+    '--session-cookie-secure', envvar='SESSION_COOKIE_SECURE', default=True
+)
+def create_app(
```

Comment:
Do the click library decorators prevent the test suite from calling `create_app()` like a normal function?